package.json

```diff
@@ -83,6 +83,7 @@
     "adm-zip": "^0.5.9",
     "body-parser": "^1.20.0",
     "commander": "^9.3.0",
+    "detect-port": "^2.1.0",
```

Comment:
maybe dumb question: couldn't you just try to listen on the port and catch the error if the address is already in use? instead of adding a new dependency.

Reply:
It's not a dumb question. That's basically what the dependency does.

![image](https://github.com/user-attachments/assets/f3e929ba-9ab6-47b9-ac32-7fba4cbe718b)